requirements.txt

```diff
@@ -1 +1,2 @@
-msrest>=0.4.4,<0.5.0
+msrest>=0.4.4
```

Comment:
why removing <0.5.0?
msrestazure 0.4 is supposed to work with msrest 0.4. For me it's important to keep the sync